### PR TITLE
improved Services.yaml example

### PR DESCRIPTION
**Pull request by franzholz**

### CODE REVIEW

File: Documentation/Configuration/Yaml/ServicesYaml.rst [hunk 1]
```diff
@@ -27,4 +27,5 @@ A typical :file:`Configuration/Services.yaml` may look like this:
    * TYPO3 uses the Symfony Dependency Injection component. The official documentation can be found at
      https://symfony.com/doc/current/service_container.html
 
-   * It is possible to use a glob pattern for the file paths, for example `'../Classes/*'`.
+   * A glob pattern is used to set the included files and their path, for example `'../Classes/**'`
```
Comment (lolli42) [inline Documentation/Configuration/Yaml/ServicesYaml.rst:30]: there should not be two stars

**Review by lolli42 — CHANGES_REQUESTED** [review]

File: Documentation/Configuration/Yaml/ServicesYaml.rst (hunk 1 above)
Comment (franzholz) [reply Documentation/Configuration/Yaml/ServicesYaml.rst:30]: See the documentation at https://en.wikipedia.org/wiki/Glob_(programming).

2 stars are needed to find all the files, and not only those from the subfolder but also from folders below the subfolders.

**Comment by garvinhicking** [comment]
Closed because of https://github.com/TYPO3-Documentation/t3docs-examples/pull/286

**Comment by franzholz** [comment]
Please reopen this issue!

**Comment by sbuerk** [comment]
> Why is this issue closed already?

Garvin mentioned it to discuss this in the other closed issue. There is no need for this changes.

https://github.com/TYPO3-Documentation/t3docs-examples/pull/286

File: Documentation/Configuration/Yaml/ServicesYaml.rst (hunk 1 above)
Comment (sbuerk) [reply Documentation/Configuration/Yaml/ServicesYaml.rst:30]: Symony uses the own Finder component which does not follow natig glob() patterns. Don't mix things here.

**Comment by franzholz** [comment]
I will open another patch.
Now it seems that no asterisk is needed at all, because all class files will always be fetched for Dependency Injection. The Symfony documentation is not clear here! And I did not yet have time yet to develop PHP code in Symfony. 

I will test it without any asterisk. Why confuse readers with an unneeded asterisk?

**Comment by sbuerk** [comment]
It is not confusing. It's a legit symfony finder pattern.

**Comment by franzholz** [comment]
> It is not confusing. It's a legit symfony finder pattern.

That is fine that you finally agree that the Symfony finder uses the Glob Pattern.

And it only works with one asterisk because all files in all subfolders will get included. But I must still test this!

**Comment by sbuerk** [comment]
> And it only works with one asterisk because all files in all subfolders will get included. But I must still test this!

Sure - that is what is wanted. As documented default. So no change needed in documentation.

**Comment by franzholz** [comment]
> The core does it so and it is tested by hunderes of functional tests and instances out in the wild

I want a new issue without any asterisk. This is not tested yet.

**Comment by franzholz** [comment]
It is only confusing to use an asterisk in the default example when it is not needed.
And I want to add the missing explanation.

**Pull request closed without merging**